Include instructions about local builds in the readme (#64)
gesellix/couchdb-prometheus-exporter: README.md

disk and data size to monitor the storage overhead. The exporter can be configured via program parameters,
environment variables, and config file.

## Build the binary

> You can find pre-build releases for different platforms at or [GitHub Releases page](https://github.com/gesellix/couchdb-prometheus-exporter/releases).

Author: ```suggestion
You can find pre-build releases for different platforms at [our GitHub Releases](https://github.com/gesellix/couchdb-prometheus-exporter/releases).
```
